corehq/apps/app_manager/suite_xml/post_process/instances.py

```diff
@@ -183,6 +183,9 @@ def get_all_instances_referenced_in_xpaths(app, xpaths):
     instances = set()
     unknown_instance_ids = set()
     for xpath in xpaths:
+        if not xpath:
```

Comment:
What's this for? I think the next line short circuits empty strings anyway. 

```py
In [1]: instance_re = r"""instance\(['"]([\w\-:]+)['"]\)"""

In [3]: instance_names = re.findall(instance_re, '', re.UNICODE)

In [4]: instance_names
Out[4]: []
```

Reply:
It's in case `xpath` is `None`